docs/en/quick-answers/troubleshooting/menu-entries-missing-in-launched-dcc.md

```diff
@@ -7,8 +7,22 @@ lang: en
 
 # I've launched Nuke/Maya/etc. from SG Desktop, but the Shotgun menu is missing entries
 
-{% include info title="Note" content="This document describes functionality only available if you have taken control over a Toolkit configuration. Please refer to the [Shotgun Integrations Admin Guide](https://support.shotgunsoftware.com/hc/en-us/articles/115000067493) or contact support if you do not have a more advanced configuration." %}
+The actions that show up in the Shotgun menu are contextually configured. That means the list of available actions may be different depending on the context you’re in.
+It's possible you may not be seeing the app your after because your in the wrong context.
```

Comment:
both instances of "your" in this sentence should be "you're".

Reply:
Done!